cmd/daemon/main.go

```diff
@@ -69,10 +69,11 @@ func NewApp(cfg *Config) (app *App, err error) {
 
 func (app *App) newAppPlayer(creds any) (_ *AppPlayer, err error) {
 	appPlayer := &AppPlayer{
-		app:         app,
-		stop:        make(chan struct{}, 1),
-		logout:      app.logoutCh,
-		countryCode: new(string),
+		app:                  app,
+		stop:                 make(chan struct{}, 1),
+		logout:               app.logoutCh,
+		countryCode:          new(string),
+		externalVolumeUpdate: make(chan float32, 1),
```

Comment:
Is the buffering needed for `externalVolumeUpdate`?

I see it is required for the logic in the driver to work, but if we move the handling of spammy events to the routine below it should not be needed.